test/test_hpyerr.py

```diff
@@ -135,6 +135,28 @@ def test_HPyErr_SetObject(self):
             mod.f(ValueError("error message"))
         assert str(err.value) == "error message"
 
+    def test_HPyErr_SetFromErrno(self):
+        import pytest
+        import errno
+        mod = self.make_module("""
+            #include <stdio.h>
+
+            HPyDef_METH(f, "f", f_impl, HPyFunc_O)
+            static HPy f_impl(HPyContext *ctx, HPy self, HPy type)
+            {
+                fseek(stdin, -200, -10);
+                HPyErr_SetFromErrno(ctx, type);
+                return HPy_NULL;
+            }
+            @EXPORT(f)
+            @INIT
+        """)
+        for type in [OSError, IOError]:
```

Comment:
on Python3, `IOError` is just an alias to `OSError`. Why do you need to test both here?

Reply:
Ah, good point. I wanted to test it with at least two exceptions, so I searched how the API is used in packages and saw two two types being used. I'll find some other suitable type